acceptance/cmd/workspace/apps/run-local/script

```diff
@@ -6,8 +6,17 @@ trace errcode $CLI apps run-local --entry-point value-from.yml 2>&1
 # so we don't need to start it in background. It will install the dependencies as part of the command
 trace $CLI apps run-local --prepare-environment --entry-point test.yml 2>&1 | grep -w "Hello, world"
 
+# Get 3 unique ports sequentially to avoid conflicts
+PORTS=$(allocate_ports.py 3 | tr -d '\r')
```

Comment:
nit: would be great to modify python script to output without \r on all platforms so that `tr -d '\r'` is not necessary

Reply:
@andrewnester The script calls `write("\n".join(...))`. Is there still a `\r` injected somewhere?